### PR TITLE
Support past video processing

### PR DESCRIPTION
**Pull request by pgrill**

### CODE REVIEW

File: libs/cv_engine.py [hunk 1]
```diff
@@ -115,9 +125,21 @@ def __process(self, cv_image):
             self.log_detail["Tracker"].append(tracker_time)
         return cv_image, tmp_objects_list, post_processing_data
 
-    def process_video(self, video_uri):
+    def process_video_file(self, video_uri, video_date: str = None):
         input_cap = cv.VideoCapture(video_uri)
-        fps = max(25, input_cap.get(cv.CAP_PROP_FPS))
+        fps = input_cap.get(cv.CAP_PROP_FPS)
```
Comment (mats-claassen) [inline libs/cv_engine.py:130]: this is overridden below so you can remove this line

File: libs/loggers/source_loggers/file_system_logger.py [hunk 2]
```diff
@@ -32,7 +32,8 @@ def save_screenshot(self, cv_image):
 
     def log_objects(self, objects, violating_objects, violating_objects_index_list, violating_objects_count,
                     detected_objects_cout, environment_score, time_stamp, version):
-        file_name = str(date.today())
+        
+        file_name = time_stamp[0: 10]
```
Comment (mats-claassen) [inline libs/loggers/source_loggers/file_system_logger.py:36]: Is this always safe to do?

File: libs/cv_engine.py (hunk 1 above)
Comment (pgrill) [reply libs/cv_engine.py]: Removed

File: libs/loggers/source_loggers/file_system_logger.py (hunk 2 above)
Comment (pgrill) [reply libs/loggers/source_loggers/file_system_logger.py:36]: Yes, the time_stamp parameter always receives the result of apply `strftime("%Y-%m-%d %H:%M:%S")` to a datetime.

**Review by mats-claassen — APPROVED** [review]

**Pull request merged**